src/_live-monitor.ts

```
@@ -87,81 +89,104 @@ export class LiveMonitor implements Disposable {
-		);
-
-		// Sort blocks
-		const sortedBlocks = this.config.order === 'asc'
-			? blocks
-			: blocks.reverse();
+				pendingOperations.push(operation);
+			});
 
-		// Find active block
-		return sortedBlocks.find(block => block.isActive) ?? null;
+			await once(rl, 'close');
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)